Userhomepage.php

```diff
@@ -0,0 +1,59 @@
+<?php
+include 'partials/nav.php';
+
+if($_SESSION['loggedin']=true){
+    echo '
```

Comment:
You cannot echo a whole HTML code, Thats not a good practice. Find alternative ways to fix this.